src/components/texts/Phrase-Word.tsx

```diff
@@ -70,38 +76,50 @@ export const Word = function ({ word, dataKey }: { word: string, dataKey:string
   const markedWords = useRecoilValue(markedwordsState);
   const wordStatus = markedWords[word.toLowerCase()];
 
-  let wordClass = 'word ';
+  let wordClass = '';
 
-  if (wordStatus) wordClass += wordStatus;
+  if (wordStatus === 'learning') {
+    wordClass = 'bg-amber-500';
+  } else if (wordStatus === 'familiar') {
+    wordClass = 'bg-yellow-500';
+  }
```

Comment:
"Learned" status doesn't get a class but should.

Reply:
I was thinking about it and really there is no distinction from a users standpoint between known words and learned words. And I feel that if we are not marking unknown words, that it doesn't make sense to mark learned words. (Which is why for now I've left it out). What do you think?

Reply:
That's why I would suggest a light gray, not too intrusive but still different from already known or ignored words. Otherwise it would not make sense to offer two buttons (learned/ignore) that visibly don't do anything different. That is confusing.